CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [[8.2.3]](https://github.com/iExecBlockchainComputing/iexec-core/releases/tag/v8.2.3) 2023-12-13
```

Comment:
```suggestion
## [[8.2.3]](https://github.com/iExecBlockchainComputing/iexec-core/releases/tag/v8.2.3) 2023-12-14
```

Reply:
Thanks for the reminder. Please see https://github.com/iExecBlockchainComputing/iexec-core/pull/644/commits/caff7a75a410b490aa55eff377668e3e19bf94f6.